server/colors.ts

```diff
@@ -0,0 +1,21 @@
+export default {
+    // Pokemon types colors
+    bug: '#A6B91A',
+    dark: '#705746',
+    dragon: '#6F35FC',
+    electric: '#F7D02C',
+    fairy: '#D685AD',
+    fighting: '#C22E28',
+    fire: '#EE8130',
+    flying: '#A98FF3',
+    ghost: '#735797',
+    grass: '#7AC74C',
+    ground: '#E2BF65',
+    ice: '#96D9D6',
+    normal: '#A8A77A',
+    poison: '#A33EA1',
+    psychic: '#F95587',
+    rock: '#B6A136',
+    steel: '#B7B7CE',
+    water: '#6390F0',
+  } as  { [key: string]: string };
```

Comment:
The `key:string` is the pokemon type, can you use it instead "random" string?

Reply:
i can't use PokemonType as the key type

Reply:
```suggestion
  } as  { [key in APIPokemonType]: string };
```

Something like this